cli/cmd/common/common.go

```diff
@@ -27,7 +27,7 @@ const (
 
 var (
 	TimescaleDBBackUpKeyForValuesYaml = []string{"timescaledb-single", "backup", "enabled"}
-	PrometheusLabels                  = map[string]string{"app": "prometheus", "prometheus": "tobs-kube-prometheus-prometheus"}
+	PrometheusLabels                  = map[string]string{"app.kubernetes.io/managed-by": "prometheus-operator", "prometheus": "tobs-kube-prometheus-prometheus"}
```

Comment:
One question, isn't the value of `prometheus` label dependant on user-provided data? As in configurable with https://github.com/timescale/tobs/blob/master/chart/values.yaml#L95 ?